regexes/device/mobiles.yml

```diff
@@ -3407,6 +3417,18 @@ LG:
       model: 'Optimus Bright L-07C'
     - regex: '(?:LG-)?VS995'
       model: 'V20'
+    - regex: '(?:LG-)?F800(?:S|K|L)?'
+      model: 'V20'
+    - regex: '(?:LG-)?H91[058]'
+      model: 'V20'
+    - regex: '(?:LG-)?H990(?:DS|N|T)?'
+      model: 'V20'
+    - regex: '(?:LG-)?LS997'
+      model: 'V20'
+    - regex: '(?:LG-)?VS995'
```

Comment:
this one seems to be a duplication.

Reply:
Do you prefer to regroup them into a single regex?

Reply:
that's not necessarily needed. but this specific regex is already done some lines before